docs/reST/tut/PygameIntro.rst

```diff
@@ -63,43 +63,18 @@ version of that same thing. This should be simple enough to follow
 along, and a complete breakdown follows.
 
 .. image:: intro_ball.gif
-   :class: inlined-right
-
-.. code-block:: python
-   :linenos:
-
-   import sys, pygame
-   pygame.init()
-
-   size = width, height = 320, 240
-   speed = [2, 2]
-   black = 0, 0, 0
-
-   screen = pygame.display.set_mode(size)
-
-   ball = pygame.image.load("intro_ball.gif")
-   ballrect = ball.get_rect()
-
-   while True:
-       for event in pygame.event.get():
-           if event.type == pygame.QUIT: sys.exit()
-
-       ballrect = ballrect.move(speed)
-       if ballrect.left < 0 or ballrect.right > width:
-           speed[0] = -speed[0]
-       if ballrect.top < 0 or ballrect.bottom > height:
-           speed[1] = -speed[1]
-
-       screen.fill(black)
-       screen.blit(ball, ballrect)
-       pygame.display.flip()
+.. image:: introsnippet.PNG
```

Comment:
I don't think the code listing should be removed for an image. One reason is the code listing can be copy/pasted, but the image not.

Can you please remove this change?

Reply:
let me know if the new edit is ok, the reasoning is, the code block doesn't show up at all on the github page, but I don't want to negatively impact https://www.pygame.org/docs/tut/PygameIntro.html, i can remove the change altogether and go back to how it was if needed

Reply:
It seems removing the linenos fixed it so it displays in github again. Thanks.